scripts/mvftoms.py

```diff
@@ -545,31 +540,38 @@ def _cp_index(a1, a2, pol):
         field_names, field_centers, field_times = [], [], []
         obs_modes = ['UNKNOWN']
         total_size = 0
-
-        # Create the MeasurementSet
-        table_desc, dminfo = ms_extra.kat_ms_desc_and_dminfo(
+        dump_start = 0
+        if os.path.exists(ms_name):
+            print(f"MS '{ms_name}' already exists - continuing...")
+            nbaselines = dataset.shape[2]/4
```

Comment:
There is already a variable called `nbl` just a few lines back that will also be more accurate (since you might have a single polarisation).